packages/angular/src/generators/setup-ssr/lib/update-project-config.ts

```diff
@@ -87,6 +87,11 @@ export function updateProjectConfig(tree: Tree, schema: Schema) {
     ];
     updateNxJson(tree, nxJson);
   }
+  if (nxJson.targetDefaults) {
+    nxJson.targetDefaults.server ??= {};
+    nxJson.targetDefaults.server.cache ??= true;
+    updateNxJson(tree, nxJson);
+  }
```

Comment:
If I look at the changes here (and in other files of this PR), if we have tasks runner options, we are going to add the target to its cacheable operations, but we are also going to have the target defaults with the `cache` option set.

Is it intentional to duplicate the configuration? Shouldn't we only have one or the other while we transition? I'd expect we:

- Set the `cache` option in target defaults for new workspaces
- Continue using `cacheableOperations` in old workspaces until we migrate them off of it

Reply:
Jason and I talked about this, there are a few reasons that it makes sense to keep cacheableOperations up to date, and we have a migration for existing repos as is so most folks shouldn't see the duplicate implementation.